scripts/script.js

```diff
@@ -42,13 +42,13 @@ HeaderToNav.prototype.buildElements = function() {
         // 'h2' headers are section categories used to group the messages.
         // 'h3' headers are the message items.
         var catClass = $(this).is('h2') ? ' class="navCategory"' : '';
-        that.addToDom += '<li' + catClass + '><a href="' + id + '">' + text + 
+        that.addToDom += '<li class="index"' + catClass + '><a href="' + id + '">' + text + 
                          '</a></li>';
     });
 
     // wrap 'li' items in 'ul' and 'nav'.
-    this.addToDom = '<nav id="side"><ul>' + 
-                    this.addToDom + '</ul></nav>'
+    this.addToDom = '<div id="frame"><nav id="side"><ul>' + 
```

Comment:
Instead of making this new element, we can target the `nav` tag.
For instance, if you want to show/hide the `nav` within `#frame`, instead you can show/hide the `ul` within `nav`